lib/web/apiserver_test_utils.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package web
+
+import (
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/gravitational/trace"
+)
+
+// NewDebugFileSystem returns the HTTP file system implementation
+func newDebugFileSystem() (http.FileSystem, error) {
+	assetsPath := "../../webassets/teleport"
```

Comment:
```suggestion
	const assetsPath = "../../webassets/teleport"
```

Note also that this makes a strong assumption about the PWD, which is likely to break for tests nested at different levels (ie, `../../` won't take it to the root). A much better solution is to chdir backwards until you hit the project root and go from there.

This looks web-specific, so it's alright, but it might warrant a comment.

Reply:
That's a good point, however this path had been used for a couple years without issues :D

Comment added.